packages/ui/src/components/Stepper/__stories__/index.stories.tsx

```diff
@@ -3,8 +3,13 @@ import { Stepper } from '..'
 
 export default {
   component: Stepper,
+  decorators: [StoryComponent => <StoryComponent />],
   title: 'Components/Navigation/Stepper',
```

Comment:
You can add `subcomponents` here and add `Stepper.Step` into it so we will see the prop documented on storybook.